src/components/structures/LeftPanel.js

```diff
@@ -73,6 +86,22 @@ const LeftPanel = React.createClass({
         return false;
     },
 
+    componentDidUpdate(prevProps, prevState) {
+        if (prevState.breadcrumbs !== this.state.breadcrumbs) {
+            Analytics.setBreadcrumbs(this.state.breadcrumbs);
+        }
+    },
+
+    _onBreadcrumbsChanged: function(settingName, roomId, level, valueAtLevel, value) {
+        // Features are only possible at a single level, so we can get away with using valueAtLevel.
+        // The SettingsStore runs on the same tick as the update, so `value` will be wrong.
+        this.setState({breadcrumbs: valueAtLevel});
```

Comment:
could also send analytics here, but componentDidUpdate will also work.

Reply:
Theory was that componentDidUpdate would catch all the places it could be updated, but with a single place it could be updated at it might not make sense... I'll leave it in componentDidUpdate for now and consider moving it when the labs flag gets replaced with a real setting.